Options.py

```diff
@@ -854,13 +855,51 @@ def __iter__(self) -> typing.Iterator[str]:
     def __len__(self) -> int:
         return self.value.__len__()
 
+    # __getitem__ fallback fails for Counters, so we define this explicitly
+    def __contains__(self, item):
+        return item in self.value
+
+
+class CounterOption(OptionDict):
```

Comment:
I just realized shouldn't this be OptionCounter (to keep the naming scheme consistent)